src/servers/src/http/prometheus.rs

```diff
@@ -600,7 +618,7 @@ pub async fn label_values_query(
             start: start.clone(),
             end: end.clone(),
             step: DEFAULT_LOOKBACK_STRING.to_string(),
```

Comment:
Lift `step` param to HTTP params can be a follow-up.

No a blocker to this patch.

Reply:
`step` is not a part of `/labels` API. We don't need to do that.